.github/workflows/sanity.yml

```diff
@@ -27,6 +27,10 @@ on:
         default: 'sanity'
         description: 'Pytest marker expression that will be used to select the tests to execute'
         required: false
+      build_url:
+        default: '0'
```

Comment:
What does a default value of 0 mean? That's pretty hard to understand. I'd prefer to default to an empty string and explain in the description what happens in this case.

Reply:
'0' is the default string is given in ini file as well but we can give any value

Reply:
Again, that won't help people using this workflow. Please change this value to something more meaningful and/or add some explanation to the description.

Reply:
done